src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,30 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
```

Comment:
Nit: we also like to have a test that is .NET Framework specific (i.e. `SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]`
This is so that we document the difference in code, but also so we know if/when .NET framework fixed this bug